### PR TITLE
Hugo Server Update - November 26, 2021

### PR DESCRIPTION
**Pull request by deeleeramone**

### CODE REVIEW

File: .github/workflows/labels.yml [hunk 1]
```diff
@@ -1,9 +1,9 @@
 name: Pull Request Labels
 "on":
-  pull_request:
```
Comment (DidierRLopes) [inline .github/workflows/labels.yml:3]: This shouldn't have been changed

File: .github/workflows/labels.yml [hunk 2]
```diff
@@ -1,9 +1,9 @@
...
+  pull_request: unlabeled
     types: [opened, reopened, labeled, unlabeled, synchronize]
 jobs:
-  label:
```
Comment (DidierRLopes) [inline .github/workflows/labels.yml:6]: Neither this

**Review by DidierRLopes — CHANGES_REQUESTED** [review]

**Comment by deeleeramone** [comment]
Ya.. my bad. I thought I was directed to the file with the flag that was missing.

**Comment by DidierRLopes** [comment]
@deeleeramone 

Can you change it back and commit? So we can merge this baby?

**Comment by deeleeramone** [comment]
Hoping I did that right! 😆

**Comment by DidierRLopes** [comment]
@deeleeramone yes! It looks good to me!

**Review by DidierRLopes — APPROVED** [review]
Btw no need to add the date on the title, it's better if you actually mention what sort of menus did you update so we can trace it back

**Pull request merged**